pkgs/development/coq-modules/multinomials/default.nix

```diff
@@ -4,7 +4,10 @@ with lib; mkCoqDerivation {
 
   namePrefix = [ "coq" "mathcomp" ];
   pname = "multinomials";
+  opam-name = "coq-mathcomp-multinomials";
```

Comment:
@Zimmi48 could you maybe use the `namePrefix` list to compose the `opam-name` automagically?